Tests/TensorFlowTests/OperatorTests/MathTests.swift

```diff
@@ -58,6 +58,76 @@ final class MathOperatorTests: XCTestCase {
         testElementaryFunction(name: "root",
                                { x in root(x, 3) }, { x in Float.root(x, 3) })
     }
+    
+    func testAbs() {
+        let x = Tensor<Float>([-1.0])
+        let y = abs(x)
+        assertEqual(y, Tensor<Float>([1.0]))
+    }
+    
+    func testSquaredDifference() {
+        let x = Tensor<Float>([-5.8])
+        let y = Tensor<Float>([-5.7])
+        let z = squaredDifference(x, y)
+        let aproxZ = 132.25
```

Comment:
Nit: I think this would be `approxZ`.

Reply:
Sorry about that! will make the changes promptly